test-example-diagrams/src/main/resources/variables/Variables.md

```diff
@@ -0,0 +1,27 @@
+# Call Activity
```

Comment:
Here, the header should be **Variables**

Reply:
Done